common/src/sql/dbinit.sql

```diff
@@ -245,18 +251,19 @@ CREATE UNIQUE INDEX ON omicron.public.silo (
  */
 CREATE TABLE omicron.public.silo_user (
     id UUID PRIMARY KEY,
-
-    silo_id UUID NOT NULL,
-
     time_created TIMESTAMPTZ NOT NULL,
     time_modified TIMESTAMPTZ NOT NULL,
-    time_deleted TIMESTAMPTZ
+    time_deleted TIMESTAMPTZ,
+
+    silo_id UUID NOT NULL,
+    external_id TEXT
```

Comment:
this is their username with the IdP, right? like their company email address?

Reply:
it could be that, it could also be some pseudo-random id - see section 8.3 of http://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf. we can't rely on this being anything legible like an email or username.

Reply:
I see. It's whatever they choose. It is not a display name.

Reply:
Although, we could say that if the type is `emailAddress` then it could be used as a display name.

Reply:
Won't we be storing other metadata about the user that will let us reliably get display names?